src/App.js

```diff
@@ -29,10 +29,28 @@ const App = () => {
         <div class="header__square-pic" />
       </header>
 
-      <div class="content">
-        <div class="description">
-          <div class="two-columns">
-            <div class="brief">Главные проблемы в обучении</div>
+      <div className="app__content">
+        <div className="content__description">
+          <div className="description__two-columns">
+            <h3 className="two-columns__brief">Главные проблемы в обучении</h3>
+            <div className="two-columns__main-text">
+              <p className="description__p">
```

Comment:
p means nothing out of HTML. Imagine you will meet this is JS document.querySelector(".description__p") if you have not written this, it will confuse you.

Use `paragraph` or `text` instead

Reply:
Same for the test of `p`s